tests/pthread_cooperation/Makefile

```diff
@@ -1,10 +1,12 @@
 include ../Makefile.tests_common
 
 BOARD_BLACKLIST := arduino-duemilanove arduino-mega2560 arduino-nano \
-                   arduino-uno jiminy-mega256rfr2 mega-xplained waspmote-pro
+                   arduino-uno jiminy-mega256rfr2 mega-xplained waspmote-pro \
+                   hifive1
 # arduino mega2560 uno duemilanove : unknown type name: clockid_t
 # jiminy-mega256rfr2: unknown type name: clockid_t
 # mega-xplained: unknown type name: clockid_t
+# hifive1: not enough memory for thread stacks
 
 BOARD_INSUFFICIENT_MEMORY := nucleo-f031k6
```

Comment:
Why not use this instead ???

Reply:
> Why not use this instead ???

I actually had it there before, but BOARD_INSUFFICIENT_MEMORY is supposed to skip the linking step if the binary would exceed either RAM or ROM. We'll soon (#11168) be making sure that this is actually the case. This app links correctly but fails at runtime...